src/PageTypes.ts

```diff
@@ -16,11 +16,13 @@ limitations under the License.
 */
 
 /** The types of page which can be shown by the LoggedInView */
-export default {
-    HomePage: "home_page",
-    RoomView: "room_view",
-    RoomDirectory: "room_directory",
-    UserView: "user_view",
-    GroupView: "group_view",
-    MyGroups: "my_groups",
-};
+enum PageType {
```

Comment:
should be able to put `export default enum PageType` on this line instead of later on where it'll get lost/forgotten.

Reply:
wtf typescript